src/k8s/pkg/utils/file.go

```diff
@@ -9,10 +9,20 @@ import (
 	"path/filepath"
 	"regexp"
 	"syscall"
-
-	"github.com/canonical/k8s/pkg/snap"
 )
 
+func Path(parts ...string) string {
+	return filepath.Join(append([]string{os.Getenv("SNAP")}, parts...)...)
+}
+
+func DataPath(parts ...string) string {
+	return filepath.Join(append([]string{os.Getenv("SNAP_DATA")}, parts...)...)
+}
+
+func CommonPath(parts ...string) string {
+	return filepath.Join(append([]string{os.Getenv("SNAP_COMMON")}, parts...)...)
+}
+
```

Comment:
Let's make these `SnapPath()`, `SnapDataPath()`, `SnapCommonPath()` instead.

On the calling sites, `util.SnapCommonPath()` would be a bit more helpful than `util.CommonPath()`, without having to lookup the implementation.